pkgs/top-level/all-packages.nix

```diff
@@ -24588,8 +24588,6 @@ with pkgs;
 
   vencord-web-extension = callPackage ../by-name/ve/vencord/package.nix { buildWebExtension = true; };
 
-  vesktop = callPackage ../by-name/ve/vesktop/package.nix { pnpm = pnpm_8; };
```

Comment:
i'd prefer this be changed to pnpm_9. alternatively we could call pnpm_9 in our arguments directly